src/test/ui/proc-macro/nonterminal-expansion.rs

```diff
@@ -0,0 +1,33 @@
+// check-pass
+// aux-build:test-macros.rs
```

Comment:
Could you add:

```rust
// compile-flags: -Z span-debug

#![no_std] // Don't load unnecessary hygiene information from std
extern crate std;
```

to this test? That will make the output print actual span numbers, which makes it much easier to determine if any future change to the stdout is legitimate or not.

Unfortunately, I don't know a better way other than copy-pasting this header into all of the tests that using `test_macros`